src/main/resources/swagger/api-docs.yaml

```diff
@@ -4943,6 +4944,103 @@ paths:
           description: Internal Error
           content: {}
       x-passthrough: false
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/importJob:
+    get:
```

Comment:
work I am _not_ taking on in this PR: return the type of import (pfb, tdrexport) in the listing of import jobs (AS-945); allow filtering of list by type of job.

Reply:
additionally, I have _not_ added integration test coverage for `POST importJob` as well as `POST importPFB`, because I didn't want to increase our test durations. Should I run each import integration test twice, once for `importJob` and once for `importPFB`?